### DISCUSSION TITLE
権限を付与していただけないでしょうか

### DISCUSSION THREAD

**Discussion by Acha0203** [sinProject-Inc/sinpro-template#15] [Q&A]
@iamsin 

[プロダクト方針対応表](https://docs.google.com/spreadsheets/d/1REZlXMy67XGSBubbESUxfwXGV54So5r7FYwZZDd69bI/edit?pli=1#gid=0)にあるテンプレートの修正に対応するため、issue を立てたのですが、権限がないため自分自身をアサインしたりラベルを付けたりできず、push もできません。権限を付与していただけないでしょうか。

**Comment by sinproject-iwasaki** [comment]
@Acha0203 

こちらのリポジトリは対応対象外ですので、、[sinpro-template-for-sveltekit](https://github.com/sinProject-Inc/sinpro-template-for-sveltekit) でよろしいでしょうか？for-sveltekit の方は、権限がついておりますのでご確認くださいませ。

また、@iamsin は予備アカウントとなっておりますので、今後は @sinproject-iwasaki のメンションを付けていただけましたら幸いです。

**Reply by Acha0203** [reply]
ご回答ありがとうございます。

こちらのリポジトリも対応する必要があると勘違いしておりました。大変失礼いたしました。各 Issue は Close いたします。
for-sveltekit の方はすでに対応済みです。
メンションの件も承知いたしました。

**Reply by sinproject-iwasaki** [reply]
@Acha0203 

テンプレート等、こちらのリポジトリも対応したほうが良さそうだと改めて思いました。
対応表の template の列にも池田さんの名前を追加しました。
そこで、権限の付与を行いましたので、今後はこちらのリポジトリも池田さんの方で管理していっていただけましたら幸いです。
対応予定にしていたPRテンプレートの更新等、直近に行っておいてくださいませ。

**Comment by Acha0203** [comment]
@sinproject-iwasaki 
権限を付与していただきありがとうございます。承知いたしました。
